docs/user-guide/release-notes/2.8.x.rst

```diff
@@ -18,6 +18,17 @@ New Features
 * The default Apache configuration has changed in order to support deferred content
   downloading. All requests to ``/pulp/repos`` are now re-written to ``/pulp/content``.
 
+* Yum metadata files and DRPMs can now be copied from one repository to another using CLI.
```

Comment:
@asmacdo I am confused if we should add commands in the release notes or mention just the new changes here. I thought we can mention the commands in the docs. What is your opinion on that?